package.json

```diff
@@ -34,10 +35,10 @@
     "qrcode.react": "^1.0.1",
     "randomcolor": "^0.6.2",
     "react": "17.0.2",
+    "react-content-loader": "^6.1.0",
```

Comment:
Lets not install a package for the skeleton loaders. We can use Tailwind CSS similar to TokenSymbol. https://tailwindcss.com/docs/animation#pulse